modules/virt-creating-interface-on-nodes.adoc

```diff
@@ -42,12 +45,25 @@ spec:
 <1> Name of the policy.
 <2> Optional: If you do not include the `nodeSelector` parameter, the policy applies to all nodes in the cluster.
 <3> This example uses the `node-role.kubernetes.io/worker: ""` node selector to select all worker nodes in the cluster.
-<4> Optional: Human-readable description for the interface.
+<4> Optional: Specifies the maximum number of nmstate-enabled nodes that the policy configuration can be applied to concurrently. This parameter can be set to either a percentage value (string), for example, `"10%"`, or an absolute value (number), such as `3`.
```

Comment:
Based on below mentioned example , it is visible that policy is never getting applied in order. Nodes are picked randomly. Should we add that also as part of this writeup? or is there any logic of picking nodes for processing ?

Reply:
IMHO, we don't need to state explicitly that the order is random, similarly as it's not stated in https://docs.openshift.com/container-platform/4.9/applications/deployments/deployment-strategies.html
where I'm sure the order is random as well.
The important thing is that the result should be the same (of course, if there is an issue during configuration and an enactment fails, different enactments may end up aborted in different runs. But in any case, the NNCP as a whole would end up being Degraded, which is the final result).

Reply:
@geetikakay If there are no other suggested changes, I would like to move this to peer review and merge.

Reply:
@sjhala-ccs sure. please proceed. Thanks @rhrazdil